src/web_app_skeleton/config/logger.cr

```diff
@@ -31,6 +31,15 @@ logger =
 
 Lucky.configure do |settings|
   settings.logger = logger
+  # By default Lucky only logs when before/after pipes halt (by redirecting, or
+  # rendering a response).
+  #
+  # Set the log level to log to log every pipe that runs.
+  #
+  # For example:
+  #
+  #    settings.action_pipe_log_level = Logger::Severity::DEBUG
+  settings.action_pipe_log_level = nil
```

Comment:
Will fail until https://github.com/luckyframework/lucky/pull/1062 is merged.

Reply:
Can we also list the other values that `action_pipe_log_level` can be?

Reply:
There are a bunch but I can link to the Logger docs that show all the possible enums!